packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -64,6 +64,8 @@ oracleAdaptersProxyPackage = "reya-oracle-adapters:1.0.1@proxy"
 oracleAdaptersRouterPackage = "reya-oracle-adapters:1.0.7@router"
```

Comment:
bump version

Reply:
omnibus version

Reply:
bumped in [9aaf1fc](https://github.com/Reya-Labs/reya-deployments/pull/259/commits/9aaf1fc35b003c1c6ade51fc6e5a865b33c16901)